### ISSUE TITLE
R Session aborted with package xlsx after upgrading R and RStudio

### ISSUE THREAD

**Issue by jpalbino** [colearendt/xlsx#102]
After first upgrading RStudio to Version 1.1.456 and later upgrading R to R version 3.5.1 -- "Feather Spray", package "xlsx" began aborting the session.
I've tried re-install packages "rJava" and "xlsx" and it didn't work.
I'm using MacOS HighSierra version 10.13.6 in a MacBook Pro Retina.

**Comment by colearendt** [comment]
Apologies for the delay in responding to this. Does this session abort still happen when you are running R inside a terminal / outside of RStudio? Is the issue still outstanding?

**Comment by jpalbino** [comment]
Hello Cole, tks for your answer!
I made a single test right now and in a terminal this problem doesn´t occur.
Only inside RStudio.

**Comment by colearendt** [comment]
@jpalbino In line with #87 , do you mind confirming if `library("rJava"); .jinit()` also causes this problem? The problem / crash could be a part of `rJava`, since that is loaded when `xlsx` is loaded.

Further, if there are certain commands that cause the crash, that would be helpful, as well! Unfortunately, I am having a hard time reproducing this.

**Comment by jpalbino** [comment]
@colearendt Hello, Cole. I'm so sorry, but the problem was in my jdk missinstallation... 
I updated my jdk to a new version, and when I called "rJava" in RStudio the problem arose!
Searching on StackOverflow (
https://stackoverflow.com/questions/30572258/error-of-java-path-on-loading-rjava-package/39059638#39059638) I found out where the problem was. Just ran "sudo R CMD javareconf" and it worked ok!
Anyway, thanks for your support!
